### DISCUSSION TITLE
Disable public IPv4 on worker nodes

### DISCUSSION THREAD

**Discussion by mohanisch** [kube-hetzner/terraform-hcloud-kube-hetzner#1200] [Q&A]
Hello!
Is it possible to disable the public ipv4 for the worker nodes? I only want to use ipv6 for them.

Cheers
Marco

**Comment by mohanisch-sixt** [comment]
Ok, found this one. 
https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/discussions/837#discussioncomment-6178928
